modules/ROOT/pages/quick-start.adoc

```diff
@@ -212,15 +213,15 @@ class CreateDinosaurIT {
 
 }
 ----
-<1> `BonitaTests` is an annotation that register a JUnit 5 extension. It's a convenient way to set up a Bonita test class. This extension allows injecting a `BonitaTestToolkit` correctly instantiated in test methods.
+<1> `BonitaTests` is an annotation that registers a JUnit 5 extension. It's a convenient way to set up a Bonita test class. This extension allows injecting a `BonitaTestToolkit` correctly instantiated in the test methods.
```

Comment:
I disagree, for me we should not add a 'the' here. 
The is a definite article, we could use it here if we were talking about a specific group of methods. But we talk about all the tests methods that users will write, so we should definitively use the indefinite form ( a / an in singular, nothing in plural). 

```suggestion
<1> `BonitaTests` is an annotation that registers a JUnit 5 extension. It's a convenient way to set up a Bonita test class. This extension allows injecting a `BonitaTestToolkit` correctly instantiated in test methods.
```